packages/user/src/migrations/Migration20240208015217.ts

```diff
@@ -0,0 +1,20 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240208015217 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table "invite" ("id" text not null, "email" text not null, "accepted" boolean not null default false, "token" text not null, "expires_at" timestamptz not null, "metadata" jsonb null, "created_at" timestamptz not null default now(), "updated_at" timestamptz not null default now(), "deleted_at" timestamptz null, constraint "invite_pkey" primary key ("id"));');
+    this.addSql('create index "IDX_invite_deleted_at" on "invite" ("deleted_at");');
+    this.addSql('create unique index "IDX_invite_email" on "invite" ("email") where deleted_at is null;;');
```

Comment:
todo: double `;` this migration can't work. Would be nice to run the migrations locally to test them everytime we generate them as a common practice